src/sage/schemes/generic/divisor_group.py

```diff
@@ -190,6 +190,36 @@ def _element_constructor_(self, x, check=True, reduce=True):
         else:
             return Divisor_generic([(self.base_ring()(1), x)], check=False, reduce=False, parent=self)
 
+    def _coerce_map_from_(self, other):
+        r"""
+        Return if there is a coercion map from ``other`` to ``self``.
+
+        There is a coercion from another divisor group if the
+        schemes are equal and there is a coercion map from
```

Comment:
```suggestion
        There is a coercion from another divisor group if there is
        a coercion map from the schemes and there is a coercion map from
```
Now this needs to be updated slightly, but this should be the last change.